config/toml.go

```diff
@@ -260,6 +260,9 @@ create_empty_blocks_interval = "{{ .Consensus.CreateEmptyBlocksInterval }}"
 peer_gossip_sleep_duration = "{{ .Consensus.PeerGossipSleepDuration }}"
 peer_query_maj23_sleep_duration = "{{ .Consensus.PeerQueryMaj23SleepDuration }}"
 
+# Block time parameters. Corresponds to the minimum time increment between consecutive blocks.
+blocktime_iota = "{{ .Consensus.BlockTimeIota }}"
```

Comment:
we should probably change this to `block_time_iota`. Also the description doesn't help distinguish it from TimeoutCommit - should be clear that this refers to the timestamp.

Also, isn't it a ConsensusParam, rather than just a config option, @milosevic ?

Reply:
I agree that description should clearly say this is only about timestamp. Yep it's a ConsensusParam (definition: ConsensusParams contains consensus critical parameters that determine the validity of blocks.).